starters/spring-web-starters/spring-web-c7/src/main/java/io/miragon/miranum/starter/spring/web/c7/CloudEventTest2Handler.java

```diff
@@ -0,0 +1,30 @@
+package io.miragon.miranum.starter.spring.web.c7;
+
+import io.cloudevents.CloudEvent;
+import io.cloudevents.core.builder.CloudEventBuilder;
+import io.miragon.miranum.connect.cloudevents.CloudEventHandler;
+import org.springframework.stereotype.Component;
+
+import java.net.URI;
+import java.util.Optional;
+import java.util.UUID;
+
+@Component
+public class CloudEventTest2Handler implements CloudEventHandler {
```

Comment:
❓ You have a Test and Test2 class: Are these meant as actual test? How do they differ from each other (besides the increased number)? 

If these are meant as actual test shouldn't they be located in the `/test `package?

Reply:
These are not real test cases and were scheduled to be removed anywhere. It was just a integration test so see if the component scan correctly registers these handlers in the cloudevents dispatcher. They only differ in their type.

Reply:
If they are removed in the near future this is perfectly fine. Maybe you can create a separate issue for this (if it is not part of an already existing issue yet) so we do not forget about this refactoring.

Reply:
Super! Thanks a lot for your effort!

Reply:
Added spring-examples module, moved "Test" Handlers there and renamed them. Will add some documentation for it